src/Cli/Program.cs

```diff
@@ -29,6 +29,8 @@ public static int Main(string[] args)
                 }
             );
 
+            DotNetEnv.Env.TraversePath().Load();
+
```

Comment:
Did we get ACK and clarify the precedence requirements of .env over system environment variables?

Reply:
Yes